### DISCUSSION TITLE
Custom domain with hostinger not checking https

### DISCUSSION THREAD

**Discussion by Astrodevil** [community/community#43328] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I added a custom domain of Hostinger to my GitHub repo, I have tried multiple times and waited for 2,3 days still HTTPS not checking.

Also whenever _I push new updates to the repo, custom domain get automatically removed_, what's the solution of this. I am using static site generator.

Here's relevant screenshots:

**Of Hostinger DNS** 
![11](https://user-images.githubusercontent.com/73425223/210624961-9506e5d7-a151-4ba3-8636-50582f6eb2ca.png)
![11a](https://user-images.githubusercontent.com/73425223/210624970-4f56d532-719c-4f5a-a88f-04f98e67a72b.png)

**Of GitHub** 
![12](https://user-images.githubusercontent.com/73425223/210625193-80e6738c-4aac-47aa-956c-9d9b24262fd1.png)

**Comment by oyeeg786** [comment]
It seems that there is an issue with the HTTPS setup for your custom domain on Hostinger with your GitHub repository. The automatic removal of the custom domain when pushing updates suggests a misconfiguration. To resolve this, ensure that the HTTPS settings are properly configured in both Hostinger and GitHub. You may need to consult the documentation or support of both platforms for a step-by-step solution. As for your website, newtelemart.com, make sure to address the HTTPS and custom domain configuration for a seamless experience. Read more [Newtelemart](https://newtelemart.com/)
